lib/sales_engine.rb

```diff
@@ -1,19 +1,32 @@
 require 'csv'
 require_relative 'merchant_repository'
 require_relative 'merchant_parser'
+require_relative 'item_repository'
+require_relative 'item_parser'
 
 class SalesEngine
+  DEFUALT_DATA_LOCATIONS = {
+                              merchant: "./data/merchants.csv",
+                              item: "./data/items.csv"
+                           }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.